variables.tf

```diff
@@ -0,0 +1,129 @@
+variable "region" {
+  description = "The region used to launch this module resources."
+  type        = string
+  default     = ""
+}
+
+variable "profile" {
+  description = "The profile name as set in the shared credentials file. If not set, it will be sourced from the ALICLOUD_PROFILE environment variable."
+  type        = string
+  default     = ""
+}
+variable "shared_credentials_file" {
+  description = "This is the path to the shared credentials file. If this is not set and a profile is specified, $HOME/.aliyun/config.json will be used."
+  type        = string
+  default     = ""
+}
+
+variable "skip_region_validation" {
+  description = "Skip static validation of region ID. Used by users of alternative AlibabaCloud-like APIs or users w/ access to regions that are not public (yet)."
+  type        = bool
+  default     = false
+}
+
+variable "create" {
+  description = "Whether to create log resources"
+  type        = string
+  default     = true
+}
+
+variable "project_name" {
+  description = "Log project name."
+  type        = string
+  default     = ""
+}
+
+variable "use_random_suffix" {
+  description = "Whether to add a suffix for project name"
+  type        = bool
+  default     = true
+}
+
+variable "store_name" {
+  description = "Log store name."
```

Comment:
description = "Log store name. If not set, a default name with prefix `sls-module-store-` will be returned."